README.md

```diff
@@ -63,19 +63,24 @@ Expanded out a few iterations:\
     $T(n) = 18T(n/18) + 9\Theta(n^5) + 3\Theta(n^5) + \Theta(n^5)$
 
 Generate a pattern:
+
 $$T(n) = 3^k * T(n/3^k) + k * \Theta(n^5)$$
```

Comment:
It's not quite right. After substituting, for example T(n/3) in T(n), your result should be probably in the form of (n^5/3^4 + n^5) in the last part of the relation.